client/lib/plans/features-list.js

```diff
@@ -423,7 +423,7 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_AUDIO_UPLOADS ]: {
 		getSlug: () => constants.FEATURE_AUDIO_UPLOADS,
-		getTitle: () => i18n.translate( 'Audio Upload Support' ),
+		getTitle: () => i18n.translate( 'Audio upload support' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.